dragonfly_doe2/cli/translate.py

```diff
@@ -0,0 +1,40 @@
+"""Dragonfly DOE2 Translation Commands"""
+import click
+import sys
+import pathlib
+import logging
+
+from .writer import model_to_inp
+from dragonfly.models import Model
+
+_logger = logging.addLevelName(__name__)
+
+
+@click.group(help='Commands for translating Dragonfly JSON files to *.inp files.')
+def translate():
+    pass
+
+
+@translate.command('model-to-inp')
+@click.argument('model-json', type=click.Path(
+    exists=True, file_okay=True, dir_okay=False, resolve_path=True)
+)
+@click.option('--name', '-n', help='Name of the output file.', default="model",
+              show_default=True
+              )
+@click.option('--folder', '-f', help='Path to target folder.', type=click.Path(
+    exists=False, file_okay=False, resolve_path=True, dir_okay=True),
+    default='.', show_default=True
+)
+def model_to_inp(df_json, name, folder):
```

Comment:
You want to pick a different name here not to overwrite the other function from the writer module.

```suggestion
def model_to_inp_file(df_json, name, folder):
```